tests/cloud_tests/delete_all_cloud_test_indexes.py

```diff
@@ -27,10 +29,18 @@ def delete_all_test_indices(wait_for_readiness=False):
             elif suffix is None:
                 indices_to_delete.append(index.index_name)
 
+    if not indices_to_delete:
+        print("No indices to delete. Exiting.")
+        return
+
+    print("Indices to delete: ", indices_to_delete)
+    print("Marqo Cloud deletion responses:")
     for index_name in indices_to_delete:
         index = client.index(index_name)
         if index.get_status()["index_status"] == marqo.enums.IndexStatus.READY:
-            index.delete(wait_for_readiness=False)
+            print(index_name, index.delete(wait_for_readiness=False))
+        elif index.get_status()["index_status"] == marqo.enums.IndexStatus.DELETING:
```

Comment:
Rather than introducing a new index state can you do a string comparison? If we need this new enum we can introduce it in a new PR